scapy/layers/sctp.py

```diff
@@ -16,6 +16,10 @@
 from scapy.layers.inet import IP
 from scapy.layers.inet6 import IP6Field
 from scapy.layers.inet6 import IPv6
+try:
+  from scapy.contrib.diameter import DiamG
```

Comment:
This should not happen: if `diameter` contribution is loaded, then it should add the layer binding. But we don't want a layer to depend on a contrib.